ISSUE_TEMPLATE/feature_request.md

```diff
@@ -0,0 +1,20 @@
+---
+name: 
+about: Suggest an idea for this project
```

Comment:
Shouldn't this contain the link and a clear indication that opening a FR on Discourse is the right thing to do?

Reply:
The "new" way of doing templates uses the `config.yml` file to determine which files and links to show. Filenames don't really mean much other than for organization. The `name:` frontmatter lines are the names of the buttons, so if a file has a blank `name:` then it won't be used/shown. It's just a sample file really in this use. If you check the test repo you can see how it all works in practice.